foundation-extensions/adobe-experience-platform-assurance/assurance-sdk-error-logs.md

```diff
@@ -156,14 +156,14 @@ D/AdobeExperienceSDK: Assurance - Timeout - Assurance did not receive deeplink t
 
 {% tab title="iOS" %}
 ```text
-[AdobeExperienceSDK DEBUG <AEPAssurance>]: Timeout - Griffon didnot receive deeplink to start griffon session. Shutting down griffon extension
+[AdobeExperienceSDK DEBUG <AEPAssurance>]: Timeout - Assurance did not receive deeplink to start Assurance session. Shutting down Assurance extension
```

Comment:
This is an actual log message from the SDK. Can you check with Prashanth if it says Assurance or still says Griffon? If it says Griffon, we should update the log message. But, want to keep this matching the specific log message for now. Let me know if the log message does still say Griffon and we can create a JIRA ticket to get that updated (and note in the JIRA ticket to also update the doc when the log message is updated).

Reply:
Oh, and, we can see if the "didnot" needs updated in the log message as well.

Reply:
OK, thanks. I sent him a note about this. I'll make any updates to this once I hear back.

Reply:
Prashanth was able to help me synchronize all the error logs with what we currently have in the SDKs. 2 images were also updated. This did include fixing the "didnot" as well. :)